wandering-warriors/main.py

```diff
@@ -0,0 +1,71 @@
+from kivy.app import App
+from kivy.properties import StringProperty
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.uix.widget import Widget
+from kivy.config import Config
+Config.set('input', 'mouse', 'mouse,multitouch_on_demand')
+
+from modules import Abacus as AbacusBase
+from modules import DrawPad
+from modules.operations import OperationsLayout
+
+
+class Calculator(Screen):
+    pass
+
+
+class Settings(Screen):
+    pass
+
+
+class TopMenu(Widget):
+    pass
+
+
+class Abacus(AbacusBase):
+    pass
+
+
+class OperationsBar(OperationsLayout):
+    pass
+
+
+class CuneiformDrawingInput(DrawPad):
+    pass
+
+
+class TopRightButton(Widget):
+    pass
+
+
+class ClearButton(Widget):
+    pass
+
+
+class HelpButton(Widget):
+    pass
+
+
+class SettingsButton(Widget):
+    pass
+
+
+class Screen:
+    def __init__(self):
+        self.sm = ScreenManager()
+        self.sm.add_widget(Calculator(name='calculator'))
+        self.sm.add_widget(Settings(name='settings'))
```

Comment:
I don't see any compelling reason for using an abbreviation here. The general recommendation for names in Python is to use descriptive names and I think `self.screenmanager` or `self.screen_manager` would have been better for readability.